### DISCUSSION TITLE
使用BoosterX 优化游戏，可能关闭了某些系统服务。无法使用该软件

### DISCUSSION THREAD

**Discussion by Apoi-Lv** [alley-rs/fluxy#172] [Q&A] [answered]
手机浏览器扫码会显示404
404: Not Found
The requested resource could not be foundThere is no more detailed explanation.

![image](https://github.com/alley-rs/alley-transfer/assets/66068121/fcd839e7-eee9-4cab-889d-db25c6842085)

**Comment by thep0y** [comment, marked as answer]
此问题与本程序无关。

猜测可能与网络防火墙有关。
